pom.xml

```diff
@@ -52,14 +52,14 @@
     <module>plugins/com.google.cloud.tools.eclipse.appengine.localserver.test</module>
     <module>plugins/com.google.cloud.tools.eclipse.appengine.newproject</module>
     <module>plugins/com.google.cloud.tools.eclipse.appengine.newproject.test</module>
+    <module>plugins/com.google.cloud.tools.eclipse.appengine.standard.java8</module>
```

Comment:
probably not. At first I didn't see the existing modules because they were out of order.